ptsprojects/mynewt/gap_wid.py

```diff
@@ -780,13 +857,58 @@ def hdl_wid_206(desc):
     return True
 
 
+def hdl_wid_208(desc):
+    btp.gap_pair()
+    return True
+
+
+def hdl_wid_209(desc):
+    return True
+
+
+def hdl_wid_224(desc):
+    btp.gap_set_mitm_off()
+    return True
+
+
+def hdl_wid_225(desc):
+    return True
+
+
+def hdl_wid_226(desc):
+    return True
+
+
+def hdl_wid_227(desc):
+    stack = get_stack()
+
+    try:
+        btp.l2cap_conn(None, None, 128)
```

Comment:
The difference here is, that in case of socket timeout special hendler returns True, and wid handler False. This difference is required to pass GAP/SEC/SEM/BV-23-C. We can either leave this as is, or add yet another handler for this test returning False.

Reply:
I think we should add another handler.

Reply:
Oh, wid 227 is only used in BV-23, so I don't think we need another handler, because this behavior is the only thing we expect.